streampipes-client-python/docs/getting-started/installation.md

```diff
@@ -0,0 +1,39 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+-->
+
+# 📚 Installation
+
+The StreamPipes Python Client is meant to work with Python 3.8 and above. Installation can be done via `pip`:
+You can install the latest development version from GitHub, as so:
+
+```bash
+pip install <NEEDS TO BE CLARIFIED>
+```
+
+---
+# 👨‍💻 Contributing
```

Comment:
@kulbachcedric We could additionally link to our DEVELOPMENT.md here. What do you think?

Reply:
Actually, I copied it from there 🙈. 
A clean link would definitely be an improvement...